libs/blocks/gnav/gnav-appLauncher.js

```diff
@@ -0,0 +1,50 @@
+import { createTag, makeRelative } from '../../utils/utils.js';
+
+const WAFFLE_ICON = '<svg xmlns="http://www.w3.org/2000/svg"  viewBox="0 0 36 36" focusable="false" aria-hidden="true" role="img" class="spectrum-Icon spectrum-Icon--sizeS"><path d="M10 10H2V3a1 1 0 0 1 1-1h7zm4-8h8v8h-8zm20 8h-8V2h7a1 1 0 0 1 1 1zM2 14h8v8H2zm12 0h8v8h-8zm12 0h8v8h-8zM10 34H3a1 1 0 0 1-1-1v-7h8zm4-8h8v8h-8zm19 8h-7v-8h8v7a1 1 0 0 1-1 1z"></path></svg>';
```

Comment:
One more thing I've noticed - it doesn't look like the links are being "localized" at the moment. Not sure if this is planned in another task or not, so just mentioning it here. 
![image](https://user-images.githubusercontent.com/115231412/201082658-7453fd2f-fd72-415e-8bd6-34362897f6da.png)

Reply:
No localization work being done with this ticket. AFAIK that is being handled by another ticket.

Reply:
ok, nice!